### PR TITLE
update tess

### PR DESCRIPTION
**Pull request by waruqi**
https://github.com/xmake-io/xmake-repo/issues/3535

### CODE REVIEW

File: packages/l/leptonica/xmake.lua [hunk 1]
```diff
@@ -15,6 +15,14 @@ package("leptonica")
     add_deps("libwebp", {configs = {libwebpmux = true}})
     add_deps("zlib", "libtiff", "libpng", "libjpeg", "giflib", "openjpeg")
 
+    on_load("windows", function (package)
+        local vs_sdkver = import("core.tool.toolchain").load("msvc"):config("vs_sdkver")
```
Comment (waruqi) [inline packages/l/leptonica/xmake.lua:19]: local msvc = package:toolchain("msvc") or toolchain.load("msvc", {plat = package:plat(), arch = package:arch()})

File: scripts/autoupdate.lua [hunk 2]
```diff
@@ -112,6 +112,9 @@ function main(pattern)
     local count = 0
     local maxcount = 5
     local instances = _get_all_packages(pattern)
+    if #instances < maxcount then
+        maxcount = #instances
+    end
```
Comment (star-hengxing) [inline scripts/autoupdate.lua:117]: 这个应该删了？

Comment (waruqi) [reply scripts/autoupdate.lua:117]: 为啥

Comment (waruqi) [reply scripts/autoupdate.lua:117]: 删了也行 我回头单独提

**Pull request merged**